controllers/talosconfig_controller.go

```diff
@@ -383,7 +384,18 @@ func (r *TalosConfigReconciler) genConfigs(ctx context.Context, scope *TalosConf
 
 	genOptions := []generate.GenOption{generate.WithDNSDomain(clusterDNS)}
 
-	secretBundle, err := generate.NewSecretsBundle()
+	if scope.Config.Spec.TalosVersion != "" {
```

Comment:
I wonder what should be our default if nothing is specified? should we do 0.8 or 0.9? machinery provides some constants, and we could add a constant for "stable" version?

Reply:
yeah, I was assuming we handle this in the machinery package, not in the provider. if none is specified, we can assume a whatever is "stable" in machinery?

Reply:
though, in saying that, I'd kind of rather us support v0.9 instead of v0.8 if nothing is specified currently. since the v0.8 config breaks AWS. so :man_shrugging:

Reply:
I'm afraid of breaking existing customers immediately, and with 0.9 graduating to release we can add more docs on specifying Talos version